OracleSOASuite/dockerfiles/README.md

```diff
@@ -87,130 +21,49 @@ https://github.com/oracle/docker-images/tree/master/OracleJava/README.md
-
-
-## Creating a container for SOA Server
-
-Start a container to launch the Managed Server from the image created. The environment variables used to run the Managed Server image are defined in the file `soaserver.env.list`. 
-
-Create an environment file `soaserver.env.list` with the below details:
-
-    MANAGED_SERVER=<managed server name, For Exp:- soa_server1, osb_server1, bpm_server1>
-    DOMAIN_TYPE=<soa/osb/bpm>
-    ADMIN_HOST=<Admin host name>
-    ADMIN_PORT=<port number where Admin Server is running>
-    
-Sample Data will look like this:
-
-    MANAGED_SERVER=soa_server1
-    DOMAIN_TYPE=soa
-    ADMIN_HOST=slc09cwi.us.oracle.com
-    ADMIN_PORT=7001
-    
-To start a docker container for SOA server you can simply call `docker run` command and passing `soaserver.env.list`. 
-
-A sample docker run command is given below:
-
-    $ docker run -i -t  --name soams --network=SOANet -p 8001:8001   --volumes-from soaas   --env-file ./soaserver.env.list oracle/soa:12.2.1.3 "/u01/oracle/dockertools/startMS.sh"
-    
-Using `--volumes-from` reuses the volume created by the Admin container. In the above `docker run` command, `soaas` is the name of the Admin server container started in the previous step and we must use the same name used for the AdminServer to start the SOA managed server. 
-
-This includes the command to tail logs to keep the container up and running.
-
->IMPORTANT: You need to wait till all the above commands are run before you can start the SOA Server.
-
-The following lines highlight when the SOA managed server is ready to be used:    
-`INFO: Managed Server is running`
-
-`INFO: Managed server has been started`
-
-Once the SOA container is created logs will be tailed and displayed to keep the container running.
-
-Now you can access
-
-   * AdminServer Web Console at http://<hostname>:7001/console  with weblogic/Welcome1 credentials.
-
-   * EM Console at http://<hostname>:7001/em  with weblogic/Welcome1 credentials.
-
-   * SOA infra Console at http://<hostname>:8001/soa-infra with weblogic/Welcome1 credentials.
-
-   * Service Bus Console at http://<hostname>:7001/servicebus with weblogic/Welcome1 credentials.
+Build the image with the -p option which will copy and install the patches into the image.
 
+$ sh buildDockerImage.sh -v 12.2.1.3 -p
 
 # License
 
 To download and run SOA 12c Distribution regardless of inside or outside a Docker container, and regardless of the distribution, you must download the binaries from Oracle website and accept the license indicated at that page.
 
 All scripts and files hosted in this project and GitHub docker-images/OracleDatabase repository required to build the Docker images are, unless otherwise noted, released under UPL 1.0 license.
 
-
 # Copyright
 
-Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2019,2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
There should be a space after the comma after 2019 and please remove "All rights reserved." We no longer include that in copyright notices in 2020.

Reply:
Corrected it.